<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-5BQ8UDAV" ignored="0" name="test1" tests="12" failures="12" timestamp="2020-11-26T10:47:01 CST" time="91.787" errors="0">
  <testcase name="test5_reg" time="5.922" classname="com.webtest.demo.Admin_Register">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.demo.Admin_Register.test5_reg(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test5_reg -->
  <testcase name="test_reg" time="9.065" classname="com.webtest.demo.Admin_Register">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.demo.Admin_Register.test_reg(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test_reg -->
  <testcase name="test_reg" time="8.961" classname="com.webtest.demo.Admin_Register">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.demo.Admin_Register.test_reg(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test_reg -->
  <testcase name="test_reg" time="8.931" classname="com.webtest.demo.Admin_Register">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.demo.Admin_Register.test_reg(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test_reg -->
  <testcase name="test_reg" time="8.945" classname="com.webtest.demo.Admin_Register">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.demo.Admin_Register.test_reg(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test_reg -->
  <testcase name="test13" time="5.78" classname="com.webtest.demo.Admin_Login">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.demo.Admin_Login.test13(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test13 -->
  <testcase name="testLogin" time="5.619" classname="com.webtest.demo.Admin_Login">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.demo.Admin_Login.testLogin(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase name="testLogin" time="5.572" classname="com.webtest.demo.Admin_Login">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.demo.Admin_Login.testLogin(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase name="testLogin" time="5.593" classname="com.webtest.demo.Admin_Login">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.demo.Admin_Login.testLogin(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase name="testFindPwd" time="5.52" classname="com.webtest.demo.Admin_Login">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.demo.Admin_Login.testFindPwd(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testFindPwd -->
  <testcase name="testFindPwd" time="5.564" classname="com.webtest.demo.Admin_Login">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.demo.Admin_Login.testFindPwd(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testFindPwd -->
  <testcase name="testFindPwd" time="5.54" classname="com.webtest.demo.Admin_Login">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.webtest.demo.Admin_Login.testFindPwd(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testFindPwd -->
</testsuite> <!-- test1 -->
